src/platform/packages/private/kbn-repo-packages/modern/package.js

```diff
@@ -223,10 +224,14 @@ class Package {
     } else if (dir.startsWith('x-pack/solutions/observability/')) {
       group = 'observability';
       visibility = 'private';
+    } else if (dir.startsWith('x-pack/solutions/chat/')) {
+      group = 'chat';
+      visibility = 'private';
     } else {
       group = this.manifest.group ?? 'common';
       // if the group is 'private-only', enforce it
-      visibility = ['search', 'security', 'observability'].includes(group)
+      //  BOOKMARK - List of Kibana solutions - FIXME we could use KIBANA_SOLUTIONS array here once we modernize this / get rid of Bazel
+      visibility = ['search', 'security', 'observability', 'chat'].includes(group)
```

Comment:
what's blocking us from using `KIBANA_SOLUTIONS`? Oh! We need to _bazelify_ the new package so that we can reference it here, correct?

Reply:
Yes, I gave it a quick try and I did not succeed 😬